library/Reporting/Timeframe.php

```diff
@@ -26,35 +26,21 @@ class Timeframe
     protected $end;
 
     /**
-     * @param int $id
+     * Transform the timeframe model to this class
```

Comment:
Since this is a factory function I would write something like `Create timeframe from the given model`.